ethereum-bytecode-database-proto/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[workspace]
+members = ["ethereum-bytecode-database-proto"]
```

Comment:
why is main folder for this service called `ethereum-bytecode-database-proto` ? maybe remove `proto`? I don't understand why you repeat it twice